doc/help/settings.asciidoc

```diff
@@ -176,7 +176,8 @@
 |<<content.user_stylesheets,content.user_stylesheets>>|List of user stylesheet filenames to use.
 |<<content.webgl,content.webgl>>|Enable WebGL.
 |<<content.webrtc_ip_handling_policy,content.webrtc_ip_handling_policy>>|Which interfaces to expose via WebRTC.
-|<<content.windowed_fullscreen,content.windowed_fullscreen>>|Limit fullscreen to the browser window (does not expand to fill the screen).
+|<<content.fullscreen.window,content.fullscreen.window>>|Set fullscreen notification overlay timeout in miliseconds.
+|<<content.fullscreen.overlay_timeout,content.fullscreen.overlay_timeout>>|Limit fullscreen to the browser window (does not expand to fill the screen).
```

Comment:
Those descriptions seem to be swapped. Did you use `scripts/dev/src2asciidoc.py` to regenerate this (see the text at the top of the file)?